website/registrations/utils.py

```diff
@@ -46,6 +51,15 @@ def get_excel_column_name(column_index):
 
     return column_name
 
+class Store():
+    def __init__(self, registration_provider):
+        self.licenses = registration_provider.licenses_acceptable.all()
+        self.subjects = registration_provider.subjects.all()
```

Comment:
It seems that `registration_provider.subjects.all()` only returns a non-empty queryset when the provider has custom taxonomies. If the provider uses bepress (default) taxonomies, this doesn't work. Instead we should use `registration_provider.all_subjects.all()`. See here: https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/provider.py#L102